input/examples/procedure-dialysis.xml

```diff
@@ -3,22 +3,19 @@
   <!--  Example adapted from MCC eCare Plan Implementation Guide Dialysis  -->
```

Comment:
Remove  <!--  Example adapted from MCC eCare Plan Implementation Guide Dialysis  --> this is irrelevant.

Reply:
Done